src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1457,14 +1469,36 @@ static bool processSubscriptions
 
     /* Send notification */
     LM_T(LmtSubCache, ("NOT ignored: %s", trigs->cacheSubId.c_str()));
-    if (processOnChangeConditionForUpdateContext(notifyCerP,
-                                                 trigs->attrL,
-                                                 mapSubId,
-                                                 trigs->reference,
-                                                 trigs->notifyFormat,
-                                                 tenant,
-                                                 xauthToken,
-                                                 fiwareCorrelator))
+
+    CachedSubscription*  cacheP = subCacheItemLookup(tenant.c_str(), trigs->cacheSubId.c_str());
+    bool                 notificationSent;
+
+    if (cacheP)
```

Comment:
In order to avoid dupliction, you suggest to duplicate the attribute vector every time?
No, we can't do that.
What we can do, I am almost 100% sure of it, is to always send it trigs->attrL.attributeV.

I left it like this to 'prepare' for different treatment between cache and non-cache.
I really think we should implement that, merge TriggeredSubscription and CachedSubscription and stop allocating freeing like we are doing now.
However, the issue already exists and no need to prepare for it in the code.
I will try to unify the two calls, using trigs->attrL.attributeV for both of them. That should work.

[ I actually planned to do that before sending the PR, but I forgot about it in the end ]